src/util/pwa.ts

```diff
@@ -0,0 +1,17 @@
+import { WebAppIcon, WebAppManifest } from '../interfaces';
```

Comment:
this is going to be a pain but can we either inline this module or move it to `webpack-contrib`, otherwise this has to be ejected too right? kind of clutters up just ejecting the config files I think?

Reply:
That's a good point; that function is simple enough that it can be inlined.